model/errors.go

```diff
@@ -0,0 +1,17 @@
+package model
```

Comment:
we try to name source files and directories in a singular form (e.g. `error.go`).